scripts/helpers.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+log() { echo "$1" 1>&2; }
+
+# Validates dev_tags for DockerHub only pushes to "hashicorppreview" org
+dev_tags_validation() {
+  if [ -n "${1}" ]; then
+    for dt in ${1}; do
+      # dev_tags can only push to the 'hashicorppreview' dockerhub org
+      echo "dt is: $dt"
```

Comment:
This should be `log` as well, or removed if not needed, to keep stdout clean. (I'll do this before merging.)